hwcomposer/hwcomposer_backend_v11.h

```diff
@@ -63,6 +99,7 @@ class HwComposerBackend_v11 : public QObject, public HwComposerBackend {
     virtual EGLNativeWindowType createWindow(int width, int height);
     virtual void destroyWindow(EGLNativeWindowType window);
     virtual void swap(EGLNativeDisplayType display, EGLSurface surface);
+    virtual void blankDisplay(int display, bool blank);
     virtual void sleepDisplay(bool sleep);
     virtual float refreshRate();
     virtual bool getScreenSizes(int *width, int *height, float *physical_width, float *physical_height);
```

Comment:
getScreenSize calls getSingleAttribute without display/screen id, which causes getSingleAttribute to default to display 0. If the device has no internal display but only an external one the screen info exposed by the Qt platform plugin reports false values?